server/src/main/java/org/opensearch/gateway/remote/RemoteClusterStateService.java

```diff
@@ -511,6 +512,10 @@ private BlobPath getManifestFolderPath(String clusterName, String clusterUUID) {
         return getCusterMetadataBasePath(clusterName, clusterUUID).add(MANIFEST_PATH_TOKEN);
     }
 
+    public Metadata getLatestClusterMetadata(String clusterName, String clusterUUID) {
+        return Metadata.builder().build();
```

Comment:
this is TODO

Reply:
yes, waiting for download PR to be merged to rebase then